src/index.html

```diff
@@ -6,9 +6,124 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link 
+      rel="stylesheet" 
+      href="./style.css"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" 
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header
+      class="header">
+      <a 
+        href="#"
+        class="logo" 
+        >
```

Comment:
if eslinter won't throw any errors I would make it in-line as well. Throughout your code
```suggestion
      <a href="#" class="logo">
```